molotov.env

```diff
@@ -1,6 +1,7 @@
 PROJECT=pageshot
 URL_SERVER=https://pageshot.stage.mozaws.net
 VERBOSE=-v
+TEST_MODULE=loadtest-read.py
```

Comment:
we'll need to split the loadtest.py into other modules, but we can do that later.
see:  Issue #16